lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -1162,7 +1163,7 @@ class EmailWidget
         progressCell.innerHTML = $progress_icon[0].outerHTML
     $removeBtn = $(_.template('<div class="remove"><i class="icon-remove-sign">
       </i> <%= label %></div>', {label: 'Remove'}))
-    $removeBtn.click =>
+    $removeBtn.click (event) =>
```

Comment:
Yeah, I DEFINITELY think there's mismatched quotes up above, because the below syntax highlighting should be throughout the file...